lms/static/karma_lms.conf.js

```diff
@@ -27,6 +27,7 @@ var options = {
     // Otherwise Istanbul which is used for coverage tracking will cause tests to not run.
     sourceFiles: [
         {pattern: 'coffee/src/**/!(*spec).js'},
+        {pattern: 'course_bookmarks/**/!(*spec).js'},
```

Comment:
Still don't love the fact that you have to manually add features in `openedx/features` to a couple of different places in the karma config in order to get tests to run for your new feature. If you didn't know about that step while you were moving something to be a new feature, it might be easy to just assume tests for your feature were passing, when in fact they weren't running at all.

Have you given any more thought to creating a separate `features` suite, @andy-armstrong?